optimum/exporters/onnx/base.py

```diff
@@ -462,14 +458,28 @@ def generate_dummy_inputs_for_validation(self, reference_model_inputs: Dict[str,
         """
         Generates inputs for ONNX Runtime using the reference model inputs. Override this to run inference with seq2seq
         models which have the encoder and decoder exported as separate ONNX files.
+
         Args:
             reference_model_inputs ([`Dict[str, Tensor]`):
                 Reference inputs for the model.
+
         Returns:
             `Dict[str, Tensor]`: The mapping holding the kwargs to provide to the model's forward function
         """
         return reference_model_inputs
 
+    def post_process_exported_models(
+        self, path: "Path", models_and_onnx_configs: Tuple, onnx_files_subpaths: List[str]
+    ):
+        """
+        Performs any model-specific post-processing on the ONNX.
+
+        Args:
+            path (`Path`):
+                Path to the directory of the stored ONNX model.
+        """
```

Comment:
Missing docstring for the two other arguments.

Reply:
addressed in adf349aada46c5f40bca52d132fe1b8ff89e8481